js/packages/phoenix-client/package.json

```
@@ -14,23 +14,24 @@
   "scripts": {
     "clean": "rimraf dist",
     "prebuild": "pnpm run clean && pnpm run generate",
-    "generate": "openapi-ts",
+    "generate": "openapi-typescript ../../../schemas/openapi.json -o ./src/lib/api/v1.d.ts",
```

```suggestion
    "generate": "openapi-typescript ../../../schemas/openapi.json -o ./src/schema.d.ts",
```
@cephalization why not just put it at the root of src?

---

alternatively could use relay syntax `__generated__`